components/org.wso2.carbon.identity.conditional.auth.functions.user/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/user/IsMemberOfAnyOfGroupsFunctionImpl.java

```diff
@@ -93,4 +165,49 @@ private List<String> getDomainAwareGroupNames(String userStoreDomain, List<Strin
         }
         return groupsWithDomain;
     }
+
+    /**
+     * Retrieve the groups claim configured for the federated IDP.
+     *
+     * @param user Authenticated user.
+     * @return  groups claim configured for the IDP.
+     */
+    private static String getGroupsClaimURIByClaimMappings(JsAuthenticatedUser user){
+
+        String groupsClaimURI = null;
+        ClaimMapping[] claimMappings = getClaimMappings(user);
+        if (claimMappings == null || claimMappings.length < 1) {
+            return null;
+        }
+
+        /* Here we get the mapping for the application roles claim because federated IDP's groups claim is mapped to
+        local application roles claim.
+         */
```

Comment:
Use //

Reply:
used this because this is  multi-line comment.